### DISCUSSION TITLE
Android and iPhone apps, no notifications, no sound,

### DISCUSSION THREAD

**Discussion by ITITIT123** [LiveHelperChat/livehelperchat#1866] [General]
Android and iPhone apps, no notification, no sound, this problem has been a long time, I have tested both apps many times, this is a serious problem, please upgrade as soon as possible, I believe many people have this problem.

**Comment by remdex** [comment]
Notifications works for all others. Please read docs how to test notifications. https://doc.livehelperchat.com/docs/mobile#live-notifications-testing

**Comment by ITITIT123** [comment]
I've tested it many times, it doesn't work, it's buggy. I can't even read your own website.

**Comment by ITITIT123** [comment]
https://demo.livehelperchat.com/site_admin/ you own this also not line, I test, do you have personally tested

**Comment by ITITIT123** [comment]
https://demo.livehelperchat.com/site_admin/ I have test, cannot use.

**Comment by remdex** [comment]
All works. Topic closed
![image](https://user-images.githubusercontent.com/1146085/210163559-c63f285c-040b-40cf-8c9e-dc69d4efdcfe.png)
